tools/accuracy_checker/accuracy_checker/representation/pose_estimation_3d_representation.py

```diff
@@ -0,0 +1,58 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .base_representation import BaseRepresentation
+from .pose_estimation_representation import PoseEstimationRepresentation
+
+
+class PoseEstimation3dRepresentation(BaseRepresentation):
+    def __init__(self, identifier='', x_values=None, y_values=None, visibility=None, labels=None,
+                 x_3d_values=None, y_3d_values=None, z_3d_values=None, fx=None):
+        super().__init__(identifier)
+        self.pose_2d = PoseEstimationRepresentation(identifier, x_values, y_values, visibility, labels)
+        self.x_3d_values = x_3d_values if np.size(x_3d_values) > 0 else []
```

Comment:
For consistency with empty  `scores` and `translation` in `PoseEstimation3dPrediction`, should be np.array([]) instead of empty lists